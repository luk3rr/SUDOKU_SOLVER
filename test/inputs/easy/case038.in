000148370 000000100 090070406 000000000 003907040 042000097 050000000 020491000 007560008 
