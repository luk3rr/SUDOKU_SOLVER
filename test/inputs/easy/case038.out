Solving the following grid:
0 0 0 | 1 4 8 | 3 7 0 
0 0 0 | 0 0 0 | 1 0 0 
0 9 0 | 0 7 0 | 4 0 6 
------+-------+------
0 0 0 | 0 0 0 | 0 0 0 
0 0 3 | 9 0 7 | 0 4 0 
0 4 2 | 0 0 0 | 0 9 7 
------+-------+------
0 5 0 | 0 0 0 | 0 0 0 
0 2 0 | 4 9 1 | 0 0 0 
0 0 7 | 5 6 0 | 0 0 8 


Solution found :')

2 6 5 | 1 4 8 | 3 7 9 
7 8 4 | 6 3 9 | 1 2 5 
3 9 1 | 2 7 5 | 4 8 6 
------+-------+------
9 7 6 | 8 2 4 | 5 3 1 
8 1 3 | 9 5 7 | 6 4 2 
5 4 2 | 3 1 6 | 8 9 7 
------+-------+------
1 5 9 | 7 8 3 | 2 6 4 
6 2 8 | 4 9 1 | 7 5 3 
4 3 7 | 5 6 2 | 9 1 8 

Algorithm: BFS
Total time: 130 ms
Total expanded states: 16970
